cloudflare/src/framework/mod.rs

```diff
@@ -45,6 +46,9 @@ impl<'a> From<&'a Environment> for url::Url {
         match environment {
             Environment::Production => {
                 url::Url::parse("https://api.cloudflare.com/client/v4/").unwrap()
+            },
+            Environment::Staging => {
+                url::Url::parse("https://api.staging.cloudflare.com/client/v4/").unwrap()
```

Comment:
Instead of coding this into the crate source, why not use `Environment::Custom`?

Reply:
for the same reason i wouldn't use `Environment::Custom` for production - it's a common use case for folks using this library and having the shorthand is convenient